churaverse-plugins-client/src/gamePlugin/domain/baseGamePlugin.ts

```diff
@@ -47,13 +39,18 @@ export abstract class BaseGamePlugin extends BasePlugin<IMainScene> {
     this.bus.unsubscribeEvent('gameEnd', this.onGameTerminate)
   }
 
+  protected get isActive(): boolean {
+    return this.gameInfoStore.games.get(this.gameId)?.isActive ?? false
+  }
+
   private getPriorGameData(ev: PriorGameDataEvent): void {
-    if (!this.getIsActive()) return
+    if (!this.isActive) return
     this.handleMidwayParticipant()
+    this.subscribeGameEvent()
```

Comment:
途中参加者は`onGameStart`が実行されない→`getPriorGameData`は途中参加の場合に受け取れる→`getPriorGameData`内でsubscribeGameEventを実行 と言う流れが若干暗黙的でぱっと見`getPriorGameData`と`onGameStart`で2回subscribeGameEventが呼ばれるように見えるので、補足コメントをつけておくと親切かなと思いました

例：
```suggestion
    // 途中参加者はonGameStartではなくgetPriorGameDataでsubscribeGameEventする
    this.subscribeGameEvent()
```